bcdaworker/worker/worker.go

```diff
@@ -62,6 +62,8 @@ func (w *worker) ValidateJob(ctx context.Context, jobArgs models.JobEnqueueArgs)
 
 func (w *worker) ProcessJob(ctx context.Context, job models.Job, jobArgs models.JobEnqueueArgs) error {
 
+	logFields := workerlog.GetLogFields(ctx)
```

Comment:
We have two log fields from `que.processJob()`'s context. Here we are using are function defined in log.go to retrieve those values so we can continue to add new fields as we encounter them.

Reply:
Ah interesting. Is it a performance optimization to avoid having an `AddLogFields(ctx, ...)` function that automatically creates a new context with the new fields, instead of having to:

1. call GetLogFields
2. update logFields
3. call WithLogFields

? I wonder if it's possible to have a nice-looking `AddLogFields` function that can accept a variable number of key-value pairs.

```
ctx = AddLogFields(ctx, {
  "file_id": fileId,
  "file_size": fileSize
})
```